IANotificationsTestProject/In App Notifications/IANAction.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+struct IANAction {
+    let message: String
+    var action: ((IANAction) -> Void)?
+    
+    init(message: String, action: ((IANAction) -> Void)? = nil) {
+        self.message = message
+        
+        if let action = action {
+            self.action = action
+        }
+    }
```

Comment:
Structs get initializers by default. You can drop L7-L11 (and it should work the same!)